src/Transformers/JsonApi/paramsSerializer.js

```diff
@@ -1,6 +1,20 @@
+import serializeFilters from './params/serializeFilters';
+import serializeParams from './params/serializeParams';
+import serializeSorting from './params/serializeSorting';
+import serializePaging from './params/serializePaging';
+import qs from 'qs';
+
 /**
- * @param {ParameterBag} params
+ * @param {ParameterBag} parameterBag
  */
-export default function paramsSerializer(params) {
-
+export default function paramsSerializer(parameterBag) {
+  return qs.stringify(
+    Object.assign(
```

Comment:
Meer een vraag: je zou ook spreading kunnen gebruiken. Misschien handig om te beslissen wanneer Object assign en wanneer spreading gebruikt wordt.

Reply:
Ik had deze bewust gekozen omdat object spread een clone maakt van het object. Ik had echter een verkeerde aanname dat Object.assign sneller zou zijn. Dat is in deze opzet dus niet zo:

https://thecodebarbarian.com/object-assign-vs-object-spread.html

Aangepast dus!